src/viking.js

```diff
@@ -1,14 +1,99 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor (health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+    
+  attack () {
+      return this.strength;
+  }
+  receiveDamage (damage) {
+    this.health -= damage;
+  }
+} 
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor (name, health, strength) {
+    super(health, strength);
+    this.name = name;
+  }
+  receiveDamage(damage) {
+     this.health -= damage;
+     if (this.health > 0) {
+       return `${this.name} has received ${damage} points of damage`
+     }else if (this.health <= 0) {
+       return `${this.name} has died in act of combat`
+     }
+  }
+  battleCry () {
+    return 'Odin Owns You All!'
+  }
+}
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier {
+  receiveDamage(damage) {
```

Comment:
Para crear e inicializar un objeto hay que hacer uso del constructor y no debe de haber mas de uno en cada clase justo como en la clase Viking 😉 
```suggestion
{
 constructor(healthArg, strengthArg) {
    super(healthArg, strengthArg);
  }
```